fmskill/plot.py

```diff
@@ -552,6 +557,30 @@ def _scatter_density(x, y, binsize: float = 0.1, method: str = "linear"):
         for i in range(len(cxy)):
```

Comment:
Is the double loop necessary?

How about?
```python
hist = histodata.flatten()
```

Reply:
I do not get the same result if I do that, also some tests fail. Double loop works :)